Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -47,12 +48,14 @@ public NavigationPage(Page root) : this()
 			PushPage(root);
 		}
 
-		public Color BarBackgroundColor {
+		public Color BarBackgroundColor
+		{
 			get => (Color)GetValue(BarElement.BarBackgroundColorProperty);
 			set => SetValue(BarElement.BarBackgroundColorProperty, value);
 		}
 
-		public Color BarTextColor {
+		public Color BarTextColor
```

Comment:
please avoid reformattings when nothing is changed

Reply:
reverted pureweens formatting fix